templates/ui-package/_github/workflows/cypress.yml

```diff
@@ -0,0 +1,76 @@
+name: Cypress Tests
+on:
+  push:
+    branches:
+      - master
+jobs:
+  cypress-run:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          # Semantic Release uses a separate Github Token for publishing
+          # See: https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md#pushing-packagejson-changes-to-a-master-branch
+          persist-credentials: false
+
+      - uses: mstachniuk/ci-skip@v1
+        # Cancels the pipeline early if [ci skip] is detected in the commit header.
+        # For example, changelog commits from the Semantic Release toolchain will generate a commit with the [ci skip] in the header.
+        with:
+          fail-fast: true
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org
+          always-auth: true
+
+      - name: Install dependencies
+        # Skip post-install scripts here, as a malicious
+        # script could steal NODE_AUTH_TOKEN.
+        run: npm i --ignore-scripts
+        env:
+          # Use the read-only access NPM token here
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+
+      - name: Build
+        # `npm rebuild` will run all those post-install scripts for us.
+        run: npm rebuild    
+
+      # Run Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        id: cypress_run  
+        continue-on-error: true
+        with:
+          install: false
+
+      - name: S3 Video Upload
+        id: s3_video_upload
+        if: always()
+        continue-on-error: true
+        run: |
```

Comment:
Adding some comments to these steps might be helpful. For example, what `date +%Y%m%d-%H%M%S | tee video-date.txt` is doing and why it's needed.